### ISSUE TITLE
The program is stuck on the dubug message

### ISSUE THREAD

**Issue by wfh1300** [platonai/exotic-amazon#9]
The web page is stuck. And The terminal occasionally prints debug messages.
**_DEBUG a.p.s.r.a.schedule.ScentRestMonitor - Try executing top N tasks ..._**
![f50a9223a68608e876a32ebc4f0e67e](https://user-images.githubusercontent.com/39584730/221103552-74cc88eb-70ff-44ea-bbfc-0f5212068e9a.png)

**Comment by platonai** [comment]
It means that all tasks have been completed.

**Comment by wfh1300** [comment]
But there are only 2,249 ASINs, which is obviously not the entire amazon data.
And the json folder is just 239MB.

**Comment by platonai** [comment]
> 

Of course, our program will not crawl the entire amazon.com by default, which would be a very huge project. To start a real world data project, you need to make some simple configuration changes. Usually, each user has some special business logic, which requires a little code modifications to meet the business needs.
